### ISSUE TITLE
Fix Prepending charachters in wallet file

### ISSUE THREAD

**Issue by mayurmarvel** [permaweb/wallet#1]
## Issue Description
The output message "Need to install the following packages: @permaweb/wallet@0.0.2 Ok to proceed? (y)" is being printed in the output file instead of the console. When I enter "y" to proceed, the output is appended to the file instead of being displayed in the console.

![appen err](https://i.ibb.co/jJddQmz/append-err.png)

**Comment by MichaelBuhler** [comment]
With this problem here, the proper way to invoke `npx` would be with the `--yes | -y flag` such as:

```bash
$ npx -y @permaweb/wallet > test1.json
```

**Comment by mayurmarvel** [comment]
Yeah, I knew that. But not everyone will use '-y' every time. So, non-techy people will get stuck with that command, and it will add complexity. That's why I proposed a solution in the pull request: https://github.com/permaweb/wallet/pull/2. It will give the user an interactive way as well as a direct way to create a wallet without any problems.
